data/json/martialarts.json

```
@@ -758,38 +758,39 @@
       {
         "id": "buff_swordsmanship_static",
         "name": "Swordsman's Stance",
-        "description": "Through chivalry and vigilance, your defense with a blade has increased.\n\n+1 Dodge attempts, blocked damage decreased by 50% of Strength.",
+        "description": "Through chivalry and vigilance, your defense with a blade has increased.\n\n+1 Block attempts; blocked damage decreased by 50% of Strength.",
         "melee_allowed": true,
-        "bonus_dodges": 1,
+        "bonus_blocks": 1,
         "flat_bonuses": [ { "stat": "block", "scaling-stat": "str", "scale": 0.5 } ]
       }
     ],
-    "onblock_buffs": [
-      {
-        "id": "buff_swordsmanship_onblock",
-        "name": "Deflection",
-        "description": "You deflected your enemy's attack and now they are open to a counterattack!\nEnables \"Sweeping Strike\" and \"Deathblow\" techniques.\nLasts 1 turn.",
-        "skill_requirements": [ { "name": "melee", "level": 1 } ],
-        "melee_allowed": true,
-        "buff_duration": 1
-      }
-    ],
-    "oncrit_buffs": [
+	"onhit_buffs": [
+	  {
+		"id": "buff_swordsmanship_onhit",
+		"name": "Conserve Momentum",
+		"description": "You maintain the momentum from your last strike to move more quickly",
+		"melee_allowed": true,
+		"skill_requirements": [ { "name": "melee", "level": 2 } ],
+		"buff_duration": 1,
+		"mult_bonuses": [ { "stat": "movecost", "scale": 0.8 } ]
+	  }
+	],	
+    "onpause_buffs": [
       {
-        "id": "buff_swordsmanship_oncrit",
-        "name": "Manslayer",
-        "description": "Your powerful attack has given you the chance to end this fight right now!\nEnables \"Vicious Strike\" techniques.\nLasts 1 turn.",
-        "skill_requirements": [ { "name": "melee", "level": 5 } ],
+        "id": "buff_swordsmanship_onpause",
+        "name": "Half Swording",
+        "description": "You grip the blade part way down for greater control. \n\n Blocked damage decreased an additional 50% of Strength, -1 Accuracy,\nEnables Grab, Mordhau and Lethal Strike",
```

```suggestion
        "description": "You grip the blade part way down for greater control. \n\n Blocked damage decreased an additional 50% of Strength, -1 Accuracy,\nEnables Grab and Lethal Strike",
```
This slipped past. I am a bit sad because a bash attack is amazing against skeletal zombies, but if you cannot make this work in the code then better cut it. My intuitive approach would have been to define a new bash damage value based on weight and length of the sword, as the current bash damage values assume the damage comes from swinging the blade while Mordhau deals damage through the pommel. But I assume it would need some C++ code to do that, if it even makes sense at all.

---

I hate to remove it too but skeletal zombies are the only ones with a big enough cut bash armor difference for mordhau to work. The jank dosent help either, the technique really just dosent work well in game. On the bright side lethal strike will help bash damage do more against skeleton zombies.